frontend/src/components/worksheets/ExtraWorksheetHTML/ExtraWorksheetHTML.js

```diff
@@ -8,13 +8,9 @@ const GlossaryModal = ({ show, toggle }) => (
             <h4>Keyboard Shortcuts</h4>
         </Modal.Header>
         <Modal.Body>
-            <a href='http://terminal.jcubic.pl/api_reference.php' target='_blank'>
-                {' '}
-                More shortcuts for web terminal (see Keyboard shortcuts section){' '}
-            </a>
             <table className='table table-striped'>
                 <tbody>
-                    <tr>Navigation</tr>
+                    <th style={{ backgroundColor: 'white', fontWeight: 'bold' }}>Navigation</th>
```

Comment:
Instead of repeating the style, can we add an identifier and define the style in css?

Reply:
th actually removes the grey background and adds font weight, so removed